docs/ai/adding-a-missing-field-to-a-terraform-resource.md

```diff
@@ -0,0 +1,42 @@
+I need to add support for a new field to the ContainerCluster KCC resource; the ContainerCluster resource uses terraform.
+
+I want to follow the example of this PR: https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/5815
+
+Please download the diff as needed (which you can do by looking at https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/5815.diff)
+
+That PR added support for this field:
+
+terraform field name: enable_cilium_clusterwide_network_policy
+terraform resource name: google_container_cluster
+KCC resource name: ContainerCluster
+KCC field path: spec.networkConfig.enableCiliumClusterwideNetworkPolicy
+
+In this case, I want to add support for this field;
+
+terraform field name: enable_nested_virtualization
+terraform resource name: google_container_cluster, google_container_node_pool
+KCC resource name: ContainerCluster, ContainerNodePool
+KCC field path: spec.nodeConfig.advancedMachineFeatures.enableNestedVirtualization
+
+Please get the correct terraform implementation from the latest terraform implementation at https://github.com/hashicorp/terraform-provider-google/blob/main/google/services/container/node_config.go
+
+As in the previous PR, please only add support for the new field to third_party/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/container/node_config.go .  Do not add support for other missing fields.
+
+Please also extend the test `pkg/test/resourcefixture/testdata/basic/container/v1beta1/containercluster/containercluster` to include setting the new field (in create.yaml) and updating the new field (in update.yaml)
```

Comment:
The test path `pkg/test/resourcefixture/testdata/basic/container/...` is hardcoded. 
Users working on different resources will need to know to change this path.